web/crux-ui/e2e/registry.spec.ts

```diff
@@ -69,7 +69,7 @@ test("Unchecked registry shouldn't search images", async ({ page }) => {
   await expect(page.locator('label[for=imageName]')).toContainText('Image name and tag')
 
   await clearInput(page.locator('input[name=imageName]'))
-  await page.locator('input[name=imageName]').type('nginx:latest:test')
+  await page.locator('input[name=imageName]').type('ghcr.io/dyrector-io/mirror/nginx:mainline-stable:test')
```

Comment:
You can use the new consts here ( eg.: `NGINX_TEST_IMAGE`), no?

Reply:
done